realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1472,6 +1472,72 @@ public long count() {
         return this.query.count();
     }
 
+    RealmResults<E> distinctAsync(final long columnIndex) {
```

Comment:
You can't do `realm.where(Dot.class).equalTo("name", "John").distinctAsync(5);` distinctAsync is package private (you can only access it UniTest) + this use the same async mechanism to deliver & update the TableView, moving it to Realm will change completely how we deliver/update async TableView & I don't see why